src/REPLICA/fix_neb.cpp

```diff
@@ -135,7 +142,8 @@ FixNEB::FixNEB(LAMMPS *lmp, int narg, char **arg) :
   else procnext = -1;
 
   uworld = universe->uworld;
-  if (NEBLongRange) {
+
+  if (NEBLongRange or EqualForceNEB) {
```

Comment:
Using "or" as logical operator is not portable (it fails to compile with Microsoft Visual C++ by default). You should be using "||". The latter form is the most common, used throughout LAMMPS, and a familiar choice in C/C++ anyway.

Reply:
Thank you for this, I was not aware, it will be changed